libcxx/test/libcxx/memory/trivial_abi/unique_ptr_destruction_order.pass.cpp

```diff
@@ -55,7 +55,7 @@ int main(int, char**) {
   func(A(shared_buf, &cur_idx), std::unique_ptr<B>(new B(shared_buf, &cur_idx)),
        C(shared_buf, &cur_idx));
 
-#if defined(_LIBCPP_ABI_MICROSOFT)
+#if defined(TEST_ABI_MICROSOFT)
```

Comment:
This test fails for the Windows CI. Can you `#include "test_macros.h"`? The other test do that.